requirements/edx/github.in

```diff
@@ -61,10 +61,11 @@ git+https://github.com/edx/django-openid-auth.git@0.15.1#egg=django-openid-auth=
 git+https://github.com/edx/MongoDBProxy.git@25b99097615bda06bd7cdfe5669ed80dc2a7fed0#egg=MongoDBProxy==0.1.0
 -e git+https://github.com/dementrock/pystache_custom.git@776973740bdaad83a3b029f96e415a7d1e8bec2f#egg=pystache_custom-dev
 -e git+https://github.com/jazkarta/edx-jsme.git@690dbf75441fa91c7c4899df0b83d77f7deb5458#egg=edx-jsme
+git+https://github.com/joestump/python-oauth2.git@master#egg=oauth2
 
 # Forked to get Django 1.10+ compat that is in origin BitBucket repo, without an official build.
 # This can go away when we update auth to not use django-rest-framework-oauth
-git+https://github.com/edx/django-oauth-plus.git@01ec2a161dfc3465f9d35b9211ae790177418316#egg=django-oauth-plus==2.2.9.edx-1
+git+https://github.com/edx/django-oauth-plus.git@01ec2a161dfc3465f9d35b9211ae790177418316#egg=django-oauth-plus==2.2.9.edx-2
```

Comment:
update the hash.

Reply:
@feanil in previous releases of `django-oauth-plus`, it seems like only the name of the release was being updated here, the hash (commit id) was from some commit in 2017, this didn't seem right, what do you think about this? , we've updated the commit id with the latest commit from the latest release.